### ISSUE TITLE
Uncaught ReferenceError: r is not defined

### ISSUE THREAD

**Issue by mrtnzlml** [relayjs/relay-devtools#37]
This extension is causing the following issue when using with refetch container: https://github.com/facebook/relay/issues/2642

Can you please check it? Thanks! :)

```text
13:47:24.877 RelayObservable.js:615 Uncaught ReferenceError: r is not defined
    at Object.unsubscribe (<anonymous>:1:2238)
    at Object.eval [as unsubscribe] (RelayObservable.js:169)
    at Object.unsubscribe (RelayObservable.js:504)
    at doCleanup (RelayObservable.js:482)
    at Object.unsubscribe (RelayObservable.js:510)
    at doCleanup (RelayObservable.js:482)
    at Object.unsubscribe (RelayObservable.js:510)
    at eval (RelayObservable.js:362)
    at Array.forEach (<anonymous>)
    at eval (RelayObservable.js:361)
```

**Comment by artola** [comment]
Same for me, using `fetchQuery` throws the error shown above.

**Comment by hatched-danny** [comment]
Getting this error as well

**Comment by tyao1** [comment]
It's fixed in #46
